content/docs/1.2.3/deploy/install/_index.md

```diff
@@ -86,14 +86,42 @@ If MountPropagation is disabled, Base Image feature will be disabled.
 
 ### Root and Privileged Permission
 
-The longhorn manager uses `Bidirectional` mount propagation to access the host's disk (default `/var/lib/longhorn`) in the container.
-An use case is to access disk metadata and replicas directory on the host.
-
-Besides, Longhorn uses `nsenter` command to execute commands on the host. The `nsenter` command, for example, is to check the host's disk space usage
-```shell
-stat /var/lib/longhorn/ -fc '{"path":"%n","fsid":"%i","type":"%T","freeBlock":%f,"totalBlock":%b,"blockSize":%s}'
-```
-The `nsenter` command rely on privileged containers and `hostPID: true`.
+Longhorn components require root access with privileged permissions to achieve volume operations and management, because Longhorn relies on system resources on the host across different namespaces, for example, Longhorn uses `nsenter` to understand block devices' usage or encrypt/decrypt volumes on the host.
+
+Below are the list of the directories of all components are:
+
+- Longhorn Manager
+  - /dev: Volume block device
+  - /proc: To access host's mount namespace by `nsenter`
+  - /var/lib/longhorn: Data path
+- Longhorn Engine
+  - /var/lib/longhorn/engine-binaries: Longhorn engine binary
+- Longhorn Instance Manager (Engine)
+  - /dev: Volume block device
+  - /proc: To access host's mount namespace
+  - /var/lib/longhorn/engine-binaries: Longhorn engine binary
+- Longhorn Instance Manager (Replica)
+  - /: Data path (allow the user to customize data path)
+- Longhorn Share Manager
+  - /dev: Volume block device
+  - /lib/modules: volume encryption by `cryptsetup`
+  - /proc: To access host's mount namespace by `nsenter`
+  - /sys: Volume encryption by `cryptsetup`
+- Longhorn CSI Plugin
+  - /: For RWX shared volume
```

Comment:
For this volume mount, here is the more detailed explanation: https://github.com/longhorn/longhorn-manager/blob/8177316cafe002b512577bb4f05a2065e80ed383/csi/node_server.go#L610

Reply:
I thought the reason is we need to support RWX volume https://github.com/longhorn/longhorn-manager/commit/a644b94439754eb791cf31955ec8b8bff8abf30b.

But if we want to grant permission to `nsenter`, shouldn't we just need to allow `/proc` mount point rather than the root mount point?

Reply:
Yeah, I am confused about this as well. The mounter is actually [this script](https://github.com/longhorn/longhorn-manager/blob/master/package/nsmounter), in which we use `-t 1` rather than something like `--net=/host/proc/1/ns/net --mount=/host/proc/1/ns/mnt`. @joshimoo Can you provide more details?

Reply:
@jenting @shuo-wu We no longer need the `/rootfs` since I have removed the nfs mounter and replaced it with the nsenter script above. Since we run the csi plugin with `hostPid: true` we can utilize `-t 1` since `/proc` maps to `/proc` in this case.

It's a left over from before where we were doing host checks via the custom mounter.

Reply:
@joshimoo So we could remove these lines https://github.com/longhorn/longhorn-manager/blob/1664880ddaedb9875671714b001a2ad7c9bc3b5c/csi/deployment.go#L525-L532 and https://github.com/longhorn/longhorn-manager/blob/1664880ddaedb9875671714b001a2ad7c9bc3b5c/csi/deployment.go#L459-L463, right?

Reply:
Then we can mention that this will be removed since v1.2.4.

Reply:
I'd prefer to add the wording that this will be removed in the coming release. 

I'm not sure if we can get this [improvement](https://github.com/longhorn/longhorn/issues/3736) into v1.2.4.
cc @innobead

Reply:
I don't think we should at this moment if it's not in 1.2.4 yet.

Reply:
Sure. Then we don't need to mention in which specific version we will remove it.